src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenPostingEditApprenticeshipDetails.cs

```diff
@@ -40,14 +42,17 @@ public async Task VerifyMapperIsCalled()
 
     public class WhenPostingEditApprenticeshipDetailsFixture : ApprenticeControllerTestFixtureBase
     {
-        public WhenPostingEditApprenticeshipDetailsFixture() : base () { }
+        public WhenPostingEditApprenticeshipDetailsFixture() : base () 
+        {
+            _controller.TempData = new TempDataDictionary( Mock.Of<HttpContext>(), Mock.Of<ITempDataProvider>());
+        }
 
         public async Task<IActionResult> EditApprenticeship(EditApprenticeshipRequestViewModel viewModel)
         {
             return await _controller.EditApprenticeship(viewModel);
         }     
 
-        public void VerifyValidationApiIsCalled()
+        public void VerifyViewModelMapperIsCalled()
```

Comment:
we have changed the name of this method but it is still verifying the api call

Reply:
Renamed it back to Api call, as there is a test for mapper already.